python/ray/rllib/ddpg/ddpg.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pickle
+import os
+
+import numpy as np
+import tensorflow as tf
```

Comment:
Would it be possible to move all of the TF code out of this file? We would like to keep the main algorithm framework agnostic